helpers/crud/cuentas_contables/comprobante.go

```diff
@@ -1,29 +1,50 @@
 package cuentas_contables
 
 import (
+	"encoding/json"
 	"fmt"
 	"net/http"
 	"strconv"
 
 	"github.com/astaxie/beego"
 	"github.com/astaxie/beego/logs"
+	"github.com/udistrital/movimientos_contables_mid/helpers"
```

Comment:
cambio menor: agregar un espacio para separar las dependencias de las que tenemos dominio (udistrital)
```suggestion

	"github.com/udistrital/movimientos_contables_mid/helpers"
```